pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1beta1
 
 import (
```

Comment:
Retry behavior: should not be different from PV/PVC and Snapshot.

@thockin: Recommended behavior: keep retrying with exponential backoff (regardless of error type) as long as the `Snapshot` object exists, and if caller doesn't want it (took too long) they can delete the `Snapshot` object. If the `Snapshot` object is deleted, stop retry, and trigger deletion. Ideally, also handle leaks -- have a storage specific object, e.g. "GCEPDSnapshot" object that is created before you start snapshot and use it to track.

Reply:
P0: investigate the behavior change.